data/processed/industry/employment_culture_entertainment_recreation.yaml

```diff
@@ -0,0 +1,30 @@
+api_doc_url: https://www.nomisweb.co.uk/api/v01/help
+api_type: POST
+data_date: 20200218
+description_short: Employment in cultural, entertainment and leisure industries
+description: This indicator measures level of employment in cultural, entertainment and leisure industries based on the Business Register Employment Survey. Those sectors are identified as clusters of SIC-4 (industry) codes
+endpoint_url: http://www.nomisweb.co.uk/api/v01/dataset/
+framework_group: place_potential
+is_experimental: True
+order: [year, nuts_id, nuts_year_spec, value.id] # USE AS IS: it specifies the order of the exported indicator fields
+region:
+  type: NutsRegion
+  level: 2
+schema:
+  region_id:
+    type: NutsRegion.id
+  region_year_spec:
+    type: NutsRegion.year_spec
+  value:
+    description: Total employment in cultural, entertainment and leisure industries
+    id: employment_culture_entertainment_recreation
+    label: Employment in cultural, entertainment and leisure industries
+    data_type: int
+  year:
+    data_type: int
+    label: Year
+source_name: ONS (https://www.nomisweb.co.uk)
+year_range: [2016, 2018]
+
+# [1] https://docs.python.org/3/library/string.html#format-specification-mini-language
```

Comment:
Minor, I think these could go, maybe we should update the README and mention them there so we don't have to delete them fromthe template?